src/main/java/com/codeborne/selenide/impl/Navigator.java

```diff
@@ -68,9 +65,6 @@ protected void navigateToAbsoluteUrl(String url, String domain, String login, St
     try {
       WebDriver webdriver = getAndCheckWebDriver();
       webdriver.navigate().to(url);
-      if (isIE() && !"".equals(login)) {
-        Selenide.switchTo().alert().authenticateUsing(new UserAndPassword(domain + login, password));
-      }
```

Comment:
@rosolko Why this code disappeared?

Reply:
@asolntsev Because selenium remove this functionality in 3.8.1 version.
From changelogs:
> Removed `Alert.authenticate` and supporting classes